src/commands/generate.ts

```diff
@@ -97,39 +97,34 @@ export const generate: Generate = async ({ rootDir, includes, useMaintainers = f
   }
 };
 
-interface Options {
-  output?: string;
+interface Options extends GlobalOptions {
   verifyPaths?: boolean;
```

Comment:
Not quite related, but is `verifyPaths` still needed? Only exists in this file, so can it be removed?

Reply:
As far as I can see, looking at the commit history, `verifyPaths` has never been used. So, it can safely be removed.

Reply:
Have removed in 53ecf24